docs/docs/tutorial-systemd.md

```diff
@@ -0,0 +1,311 @@
+
+### Step 1: Launch a Mesosphere cluster on a systemd platform
+
+The first step is to create a Mesosphere cluster on a platform that supports systemd.   While all linux distros are moving in the direction of supporting systemd, the currently released list can be boiled down to RHEL 7, Suse v12 and CoreOs.  Mesosphere's DCOS is installed on CoreOS, however in this case Mesos-DNS is pre-installed.   This tutorial assumes that you are installing a fresh Mesosphere cluster without using Mesos-DNS.  For help installing please refer to [https://docs.mesosphere.com/getting-started/datacenter/install/](https://docs.mesosphere.com/getting-started/datacenter/install/).
+
+This tutorial assumes the following cluster topology:
+
+* One Master node which includes:
+	* mesos-master
+	* zookeeper
+	* marathon
+	* IP 
+* Several slave nodes which have mesos-slave running on them.
+
+For this tutorial we will assume the master IP address is `10.14.245.208`.
+
+### Step 2: Build and install Mesos-DNS
+
+We will build and install Mesos-DNS on node `10.14.245.208`.  Access the node through ssh.
+
+You will need git for this process.   For SUSE this means `sudo zypper install git-core`
+
+The build process includes installing `go`:
+
+```
+wget https://storage.googleapis.com/golang/go1.4.linux-amd64.tar.gz
+tar xzf go*
+sudo mv go /usr/local/.
+export PATH=$PATH:/usr/local/go/bin
+export GOROOT=/usr/local/go
+export PATH=$PATH:$GOROOT/bin
+export GOPATH=$HOME/go
+go get github.com/tools/godep
+export PATH=$PATH:$GOPATH/bin
+```
+
+Now, we are ready to compile Mesos-DNS:
+
+```
+go get github.com/mesosphere/mesos-dns
+cd $GOPATH/src/github.com/mesosphere/mesos-dns
+make all
+cd
+sudo mv ~/go/bin/mesos-dns /usr/bin/mesos-dns
+```
+
+In the `/etc/mesos-dns/` directory lets create a file named `config.json` with the following contents: 
+
+```
+sudo mkdir /etc/mesos-dns/
+
+$ cat /etc/mesos-dns/config.json 
+{
+  "zk": "zk://10.14.245.208:2181/mesos",
+  "refreshSeconds": 60,
+  "ttl": 60,
+  "domain": "mesos",
+  "port": 53,
+  "resolvers": ["169.254.169.254","10.0.0.1"],
+  "timeout": 5,
+  "email": "root.mesos-dns.mesos"
+}
+```
+The `resolvers` field includes the two nameservers listed in the `/etc/resolv.conf` of the nodes in this cluster. 
+
+### Step 3: Running under systemd
+
+In the `/etc/systemd/system` directory lets create a file named mesos-dns.service with the following contents:
+
+```
+[Unit]
+Description=Mesos-DNS
+After=network.target
+Wants=network.target
+
+[Service]
+ExecStart=/usr/bin/mesos-dns -config=/etc/mesos-dns/config.json
+Restart=on-failure
+RestartSec=20
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Now start the service:
+
+`sudo systemctl start mesos-dns`
+
+### Step 4: Configure cluster nodes
+
+Next, we will configure all nodes in our cluster to use Mesos-DNS as their DNS server. Access each node through ssh and execute: 
+
+
+```
+sudo sed -i '1s/^/nameserver 10.14.245.208\n /' /etc/resolv.conf
```

Comment:
This works well if your nodes are configured with static ip. If they use dhcp, then dhclient (or something similar) will probably blow away this change at some point. For dhclient, you can modify dhclient.conf:
```
prepend domain-name-servers 10.14.245.208;
```